server/src/test/scala/org/http4s/server/middleware/CORSSpec.scala

```diff
@@ -71,9 +78,8 @@ class CORSSpec extends Http4sSpec {
 
     "Respond with 403 when origin is not valid" in {
       val req = buildRequest("/bar").replaceAllHeaders(Header("Origin", "http://blah.com/"))
-      cors2.orNotFound(req).map((resp: Response) => resp.status.code == 403)).unsafeRun
+      cors2.orNotFound(req).map((resp: Response) => resp.status.code == 403).unsafeRun
```

Comment:
I can't find this behavior in the spec, but as @haripriyamurthy and I discussed, this seems like the right thing to do and exists in other popular implementations (e.g., Tomcat's CORSFilter).